cassandra/docker-compose-auditor.yml

```
@@ -0,0 +1,43 @@
+include:
+  - ./docker-compose-ledger.yml
+  - ../common/docker-compose-auditor.yml
+
+services:
+  scalardl-auditor-schema-loader:
+    image: ghcr.io/scalar-labs/scalardl-schema-loader:${SCALARDL_VERSION}
+    environment:
+      - SCHEMA_TYPE=auditor
+    volumes:
+      - ./auditor.properties:/scalardb.properties
+    depends_on:
+      cassandra:
+        condition: service_healthy
+    command:
+      - "-c"
+      - "/scalardb.properties"
+      - "--coordinator"
```

```suggestion
    command:
      - "-c"
      - "/scalardb.properties"
```
In the ScalarDL Samples, we will use the same database for both Ledger and Auditor, so this suggestion might not have meaning.

However, strictly, I think ScalarDL Auditor uses only the storage API instead of the transactional API of ScalarDB.

Therefore, if I understand correctly, I think we don't need to specify the `--coordinator` here.

To provide the correct information to users as long as possible, it might be better to remove the `--coordinator` option from here.

What do you think?

---

You are right. It's unnecessary. Fixed in [e923d35](https://github.com/scalar-labs/scalardl-samples/pull/83/commits/e923d3590969c5242b83e2b2d29209efad509cee). Good catch, thank you!